### PR TITLE
Add Lambda function for crow species data call

### PR DESCRIPTION
**Pull request by etcadinfinitum**
:fries: :corn: :popcorn:

### CODE REVIEW

File: lambda_species/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import json
+import boto3
+
+def lambda_handler(event, context):
+	# Create a connection to the DynamoDB instance of AWS
+	dynamodb = boto3.resource('dynamodb')
+	
+	# Create a connection to the specific table we want from our DynamoDB. In this case, 'CrowFacts'.
```
Comment (anishprasad01) [inline lambda_species/lambda_function.py]: Change comment to match what's in the code
```suggestion
	# Create a connection to the specific table we want from our DynamoDB. In this case, 'CrowFactsTwo'.
```

File: lambda_species/lambda_function.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	
+	response = None
+	code = 200
+	
+	# Check that the incoming request has the necessary parameters. 
+    # In this case we are expecting 'CrowSpecies' and 'habitat'. 
+	# For a GET method, the incoming parameters need to be the primary/sorting keys of the database.
+	# For a POST method, the incoming parameters need to be the new data you want to add to the database
+	
+	if 'CrowSpecies' in event and 'location' in event:
```
Comment (anishprasad01) [inline lambda_species/lambda_function.py]: This doesn't match. If you want to look for `CrowSpecies` and `habitat`, then no need to check for `location`, which is a reserved word anyway
```suggestion
	if 'CrowSpecies' in event and 'habitat' in event:
```

**Review by anishprasad01 — CHANGES_REQUESTED** [review]
Looks good overall. Just a couple of inconsistencies to address.

File: lambda_species/lambda_function.py (hunk 2 above)
Comment (etcadinfinitum) [reply lambda_species/lambda_function.py]: LOL, you're totally right, I caught this upstream after opening the PR and thought to myself "I should fix this in a commit"

File: lambda_species/lambda_function.py (hunk 1 above)
Comment (etcadinfinitum) [reply lambda_species/lambda_function.py]: Yes TY, copy pasta strikes again

**Review by anishprasad01 — APPROVED** [review]

**Pull request merged**